### ISSUE TITLE
Running AppSource Validation with AppSourceCop - Missing File

### ISSUE THREAD

**Issue by EasyEdu-dev** [microsoft/navcontainerhelper#1552]
**Describe the issue**
Iam trying to run ALValidation with AppSourceCop to get my .app ready for AppSource. ALValidation and AppSourceCop is starting perfectly, but AppSourceCop throws an error that a certain file is missing. I dont understand atm what this file is doing and how to fix the error. Please help. 

**Scripts used to create container and cause the issue**
```
![image](https://user-images.githubusercontent.com/74726860/102493440-79929b80-4073-11eb-9fd1-b7f1da5c643c.png)


```

**Full output of scripts**
```
  _____                   _                                    _____                           _____            
 |  __ \                 (_)                 /\               / ____|                         / ____|           
 | |__) |   _ _ __  _ __  _ _ __   __ _     /  \   _ __  _ __| (___   ___  _   _ _ __ ___ ___| |     ___  _ __  
 |  _  / | | | '_ \| '_ \| | '_ \ / _` |   / /\ \ | '_ \| '_ \\___ \ / _ \| | | | '__/ __/ _ \ |    / _ \| '_ \ 
 | | \ \ |_| | | | | | | | | | | | (_| |  / ____ \| |_) | |_) |___) | (_) | |_| | | | (__  __/ |____ (_) | |_) |
 |_|  \_\__,_|_| |_|_| |_|_|_| |_|\__, | /_/    \_\ .__/| .__/_____/ \___/ \__,_|_|  \___\___|\_____\___/| .__/ 
                                   __/ |          | |   | |                                              | |    
                                  |___/           |_|   |_|                                              |_|    
Unexpected error while validating app. Error is: Ein Objekt im angegebenen Pfad "C:\Users\EA420~1.PEN" ist nicht vorhanden.
bei <ScriptBlock>, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Sort-AppFilesByDependencies.ps1: Ze
ile 40
bei Sort-AppFilesByDependencies, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Sort-AppFilesByDepend
encies.ps1: Zeile 28
bei Run-AlCops, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Run-AlCops.ps1: Zeile 77
bei <ScriptBlock>, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Run-AlValidation.ps1: Zeile 394
bei <ScriptBlock>, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Run-AlValidation.ps1: Zeile 310
bei <ScriptBlock>, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Run-AlValidation.ps1: Zeile 287
bei Run-AlValidation, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Run-AlValidation.ps1: Zeile 285
bei <ScriptBlock>, <Keine Datei>: Zeile 1
...
```

Thank you a lot.

**Comment by freddydk** [comment]
I cannot see the script / the image.
This is likely caused by a wrong parameter value, so I would need to se how you call the function?

**Comment by EasyEdu-dev** [comment]
`$validationResults = Run-AlValidation `
    -licenseFile "license.flf" `
    -previousApps @( "" ) `
    -apps @( "localpath/file.app" ) `
    -validateCurrent `
    -countries "de" `
    -affixes "xyz" `
    -supportedCountries "de"`

**Comment by freddydk** [comment]
Reformatted your message.
If you don't have any previousApps, you should leave out the parameter.
Is localpath/file.app an absolute path?

**Comment by EasyEdu-dev** [comment]
Its like : C:\dir dir\Dir2\appfile1.17.201208.0.app
I think this is a relative path.

**Comment by freddydk** [comment]
Needs to be absolute like:

C:\dir\Dir2\appfile1.17.201208.0.app

**Comment by EasyEdu-dev** [comment]
I have removed all spaces in the path. Error is still the same after rerunning the script with updated filepaths.

**Comment by freddydk** [comment]
I need the full script and the full output - if you cannot print it here, you can email it to freddyk at microsoft dot com

**Comment by freddydk** [comment]
I think this is caused by this issue: https://superuser.com/questions/1524767/powershell-uses-the-short-8-3-form-for-envtemp
I cannot repro this, but given this message, I have changed all occurances of $env:temp to a function, which doesn't used the 8-3 notation.
I will check this in and a new preview version should be available later this afternoon with this fix.

**Comment by freddydk** [comment]
I will post a message here once a preview version is available and close it when a released version is ready

**Comment by freddydk** [comment]
Shipped in 1.0.17
